recipes/libmaxminddb/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os.path
+
+from conans import ConanFile, CMake, tools
+
+
+class libmaxminddbTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "pkg_config"
+
+    def build_requirements(self):
+        self.build_requires("pkgconf/1.7.3")
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            self.run(os.path.join(".", "test_package"), run_environment=True)
```

Comment:
```suggestion
            bin_path = os.path.join("bin", "test_package")
            self.run(bin_path, run_environment=True)
```

Reply:
All right. I missed `conan_basic_setup()` in the CMake. This is now fixed.